packages/contracts/contracts/interpreters/TwoPartyEthAsLump.sol

```diff
@@ -0,0 +1,41 @@
+pragma experimental "ABIEncoderV2";
+
+import "../interfaces/Interpreter.sol";
+
+
+contract TwoPartyEthAsLump is Interpreter {
+
+  enum Resolution {
+    SEND_TO_ADDR_ONE,
+    SEND_TO_ADDR_TWO,
+    SPLIT_AND_SEND_TO_BOTH_ADDRS
+  }
+
+  struct Params {
+    address payable[2] playerAddrs;
+    uint256 amount;
+  }
+
+  function interpret(
+    bytes memory encodedResolution, bytes memory encodedParams
+  ) public {
+
+    Params memory params = abi.decode(encodedParams, (Params));
+    Resolution resolution = abi.decode(encodedResolution, (Resolution));
+
+    if (resolution == Resolution.SEND_TO_ADDR_ONE) {
```

Comment:
I see you used enums here but integer literals in the above place where this code is used that I commented on. Stick with enums?